aten/src/ATen/core/dispatch/Dispatcher.h

```diff
@@ -281,13 +283,54 @@ class CAFFE2_API OperatorHandle final {
 
 namespace detail {
 template<class... Args> inline void unused_arg_(const Args&...) {}
+
+template <typename T, std::enable_if_t<c10::impl::not_ok_to_box<T>::value>* = nullptr>
+inline void push_ivalue_copy(std::vector<c10::IValue>& stack, const T& v) {}
```

Comment:
it's kind of shitty that we just skip the arg and it makes looking at results weird. How about you push a string literal `'<cannot box it>'`

Reply:
in this case every observer needs to explicitly know about this special value and filter it out;
I will rename the function to make it more obvious that it doesn't convert all arguments

Reply:
to be more clear, arguments that are not boxable are not only unsupported in observers but also in some sense less interesting since observers wouldn't be able to do much with them anyway even if we could pass them, so I start from the assumption that observers want to receive only the ivalues they know how to handle

Reply:
Hm, still I think it's better to pass the same number of positional arguments as in schema to observer than to drop some of them. I literally mean passing `<can't box>` as a string literal, imho it's a better failure mode